### ISSUE TITLE
"/value" instead of "/value_raw"

### ISSUE THREAD

**Issue by ohmar** [neuralfraud/grafana-prtg#68]
Hello,

Is there a way to specify which value to retrieve from the returned query? I have a sensor in PRTG that divides the retrieved value by 100. That value is stored in "<value ..." and the original value is stored in "<value_raw ...". Unfortunately, I don't see a way to manipulate this value in Grafana.

**Comment by neuralfraud** [comment]
The value isn’t numeric and therefore cannot be used. So …no. You cannot do that.

**Comment by neuralfraud** [comment]
I've updated the development branch with a new multiplier function. To divide by 100, update the query and enter ".01" to the multiplier field. Let me know if that solves your problem!

Copy the "dist" directory from the development branch into your existing plugin's install directory, overwriting the directory. Restart grafana.

**Comment by ohmar** [comment]
Perfect! Thank you so much! Exactly what I was looking for.

**Comment by timuralk** [comment]
Hi,

I do not understand where we are performing these steps. I do not understand where we are performing these steps. Can you write in more detail?

"I've updated the development branch with a new multiplier function. To divide by 100, update the query and enter ".01" to the multiplier field. Let me know if that solves your problem!
Copy the "dist" directory from the development branch into your existing plugin's install directory, overwriting the directory"

Thanks for help,

**Comment by timuralk** [comment]
<img width="153" alt="grafana3" src="https://user-images.githubusercontent.com/34305356/33662199-06bb3828-da9c-11e7-8329-07e0d4779230.PNG">
<img width="114" alt="prtg" src="https://user-images.githubusercontent.com/34305356/33662201-06e31a6e-da9c-11e7-8e1a-08f63bce2c63.PNG">

**Comment by JohnH77** [comment]
@timuralk 
on my example i have download the dist directory and overwriting the dist directory in my grafana path. 
Before i have the SNMP Traffic Sensor with MByte/s in Grafana because prtg raw Traffic Data is in Bytes/s

**before**
![image](https://user-images.githubusercontent.com/30649081/33663004-7d6a4f94-da8e-11e7-8d20-76cf199d06ee.png)
![image](https://user-images.githubusercontent.com/30649081/33663007-83b45318-da8e-11e7-9dfc-b4f729a2adcc.png)

**after**
![image](https://user-images.githubusercontent.com/30649081/33663016-91d95862-da8e-11e7-94b6-fdb3a051376a.png)
![image](https://user-images.githubusercontent.com/30649081/33663060-bbfd6c28-da8e-11e7-8561-83adbe469b98.png)

**Comment by JohnH77** [comment]
in your example you have to set the Multiplier to ".01" i think. 
And under the Options you can set the Decimals to 1

**Comment by timuralk** [comment]
Hi,

This function is not available.

<img width="799" alt="graph" src="https://user-images.githubusercontent.com/34305356/33664830-0513e07e-daa6-11e7-9312-7bd74c819387.PNG">

<img width="781" alt="graph2" src="https://user-images.githubusercontent.com/34305356/33664870-2452a0e2-daa6-11e7-81e7-178e6c160c61.PNG">

**Comment by JohnH77** [comment]
Hi, 

have you download the actuall Plugin, copy and restart Grafana?

**Comment by timuralk** [comment]
Which plugin I need to load? I have updated all attachments :( Grafana PRTG plugin version 4.0.3.

**Comment by JohnH77** [comment]
you can just copy and overwrite the "dist" directory to your grafana Installation. Please restart the Server and refresh the Grafana in your Browser

![image](https://user-images.githubusercontent.com/30649081/33665157-3e226166-da96-11e7-8b83-f38f29161ee6.png)
![image](https://user-images.githubusercontent.com/30649081/33665191-5e36d342-da96-11e7-8391-40a9a1c4e2c3.png)

**Comment by JohnH77** [comment]
what Grafana Version do you have?

**Comment by timuralk** [comment]
I did this today. Now I try again, there has not been a change. I cleaned up the browser cache.  Grafana v4.6.2 (commit: 8db5f08)

**Comment by JohnH77** [comment]
I have the Same Version. 
I think you have the wrong "dist" Folder. Please Try download the Development Download.

**Comment by timuralk** [comment]
You are right, I did not download the "development file" so I experienced this problem. But I want to add another note; There was no change when I updated the "dist" file, but this feature worked when I updated the src file. Thank you very much for your support.

**Comment by stalley** [comment]
@neuralfraud Can you elaborate on the below.

> The value isn’t numeric and therefore cannot be used. So …no. You cannot do that.

The plugin is pulling data from an SNMP probe and selecting value_raw by default. Are you saying that there's no way to select just value?

`    value channel="Traffic Out (speed)" channelid="1">339,071 kbit/s</value>`
`     value_raw channel="Traffic Out (speed)" channelid="1">42383922.1130< /value_raw>`

**Comment by neuralfraud** [comment]
In Github, Click the branch button:

 



 

Then from the branches menu, select “Development”

 

Then download 😊

 

Overwrite the “Dist” directory – grafana tries this by default 

 

Then restart grafana server.

**Comment by stalley** [comment]
@neuralfraud  Thank you, I misunderstood the nature of the conversion of my data. I copied over the development branch and I do now have the multiplier option by don't see any changes in the graph or data when I apply a multiplier value.

**Comment by stalley** [comment]
@neuralfraud After logging out and logging back in to Grafana everything seems to be working. Thank you for the assistance and the great plugin.

**Comment by wlolato** [comment]
@neuralfraud This works for linux ?
